lib/parser.js

```diff
@@ -92,7 +92,7 @@ function parseLinks(description/*: Cheerio*/) {
  *
  * @param {Store} store
  */
-async function requestAndParse(store/*: Store*/) {
+async function requestAndParse(store/*: Store*/, hacketyhack/*: 'Field' | 'Parameters' */) {
```

Comment:
😆

Reply:
Yeah I just tried to get the parser going :smile:

This was funky to run on the API website in developer tools console:

```js
Array.from(document.body.querySelectorAll('body table')).map(table => [table, table.querySelector(':scope tr:first-child td:first-child')]).filter(([_, el]) => el.innerText === 'Parameters').map(([table, el]) => {

let elTry = table;
for (let i = 0; i < 50; i++) {
  elTry = elTry.previousElementSibling;
  if (elTry === null) { return null; }
  if (elTry.nodeName === 'H4') { return elTry; } 
}

}).map(heading => heading.innerText)
```

I wanted to see if the same approach of finding names would work, and seems like it does :smile: